cfme/tests/v2v/test_v2v_migrations.py

```diff
@@ -479,3 +481,55 @@ def _is_migration_started():
     for vm in vms:
         soft_assert(request_details_list.is_successful(vm) and
          not request_details_list.is_errored(vm))
+
+
+def test_migration_long_name(request, appliance, v2v_providers, host_creds, conversion_tags):
+    """Test to check VM name with 64 character should work"""
+    source_datastores_list = v2v_providers[0].data.get('datastores', [])
+    source_datastore = [d.name for d in source_datastores_list if d.type == 'nfs'][0]
+    # Following code will create vm name with 64 characters
+    vm_name = random_vm_name(context="v2v") + fauxfactory.gen_alpha(51)
```

Comment:
Required:
please string addition not good thing as per our standard please use `.format`